SEMain/src/program/SourceXtractor.cpp

```diff
@@ -88,7 +89,7 @@ namespace fs = boost::filesystem;
 using namespace SourceXtractor;
 using namespace Euclid::Configuration;
 
-static long config_manager_id = getUniqueManagerId();
+static long config_manager_id = 0L;
```

Comment:
:question:

Reply:
This was to avoid having to propagate a bazillion configuration parameters through lots of layers just for testing.

If we use config_manager_id = 0 rather than random value, we can access the configuration manager as a singleton. Which means those configuration parameters can be accessed where they are needed.

I will revert for now, it's not needed anymore.